docuilib/src/theme/Footer/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
```

Comment:
I notice the middle part of the Footer is not stretched like in Alexey's design

Reply:
Not sure I understand... what's the difference?

Reply:
It looks ok now, before it looks different